### ISSUE TITLE
User Story : I can search for a city

### ISSUE THREAD

**Issue by carinadiesel-dev** [carinadiesel-dev/weather-react#1]
### Users should be able to search for their city by typing in the name of the city and then clicking on the "Search City" button.

![image](https://github.com/carinadiesel-dev/weather-react/assets/106066035/e2b38c2d-02c8-42c6-a737-2d4f75890291)

**To do :**

- [ ] Implement the [geocoding API](https://openweathermap.org/api/geocoding-api) to search for a city
- [ ] Create area in UI where the location options will be displayed and the user can click
- [ ] Store latitude and longitude for use with weather API

**Comment by schalkneethling** [comment]
@carinadiesel-dev - This is a good start! 🎉 I would rephrase the user story as, "As a user, I want to search for a city". The other piece that is missing here is the acceptance criteria. This is less about the tasks from a developer or designer's perspective and more about defining the definition of done.

How would you know that this user story is done? Here is a good article: https://resources.scrumalliance.org/Article/need-know-acceptance-criteria

The main takeaways are:

> In this way, the user story describes the “why” of the work, while the acceptance criteria describe the “what.” The “how” is decided by developers as they work through the sprint.

Pretend you are the product owner; this user story has been marked ready for testing. What would you do to confirm that this solves the user's need? Do you think this user story is a little too atomic? Perhaps it is not, but the outcome of the user story is a little unclear.

For example, is the user story complete if there is a component on the page, as shown in the screenshot? If not, then the acceptance criteria will help to make this clear and help the development and design team understand the scope of the user story.

It might be that the outcome of the user story should be that the user can open the web app, enter the city they are looking for, press search, and is shown a list of possible matches.
